src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToControlGeneratedAssemblyInfo.cs

```diff
@@ -370,6 +370,110 @@ public void It_includes_internals_visible_to()
             AssemblyInfo.Get(assemblyPath)["InternalsVisibleToAttribute"].Should().Be("Tests");
         }
 
+        [Fact]
```

Comment:
This was originally a theory with 3 test cases, but that seems to fail in CI with the following error:

```
/private/tmp/helix/working/AE680940/w/A5DF08DC/e/testExecutionDirectory/It_includes_r---61EE9334 already exists
```

I'm guessing that is because each test tries to create a tmp directory while running.

Reply:
Yes, typically we handle this by passing in the theory parameters as the `identifier` argument to `CopyTestAsset` or similar methods.